### ISSUE TITLE
What license is this repo under?

### ISSUE THREAD

**Issue by Persistent13** [VADOSWARE/pg_idkit#16]
TL;DR What license is this repo under?

---

I'm curious, what license this repo is under? It doesn't seem to have one at the moment.

All of the crates, aside from one, are licensed MIT. The lone uuid7 create is licensed under Apache 2. I am not a lawyer and so from my limited understanding, Apache 2 and MIT licenses are compatible. You could choose to license this repo under either MIT or Apache 2.

My goal is to see if OVH would include this extension in their [suite of approved extensions](https://help.ovhcloud.com/csm/pl-public-cloud-databases-postgresql-extensions?id=kb_article_view&sysparm_article=KB0049328). They wouldn't do so without a license that allows them to do so.

Thanks for making this, it's really helpful!

**Comment by t3hmrman** [comment]
Thanks for pointing this out, it looks like I overlooked placing a license in here, thi sis a crucial oversight. 

I'd *like* to release this under the postgres license but MIT an Apache 2 are also fine -- will get this fixed within the next 15 mins.

**Comment by Persistent13** [comment]
I only mention MIT or Apache 2 as I have a vague understanding they're compatible. But really, the license choice is up to you.

Thank you for taking the time to consider this request, Victor!

**Comment by t3hmrman** [comment]
Yeah, Apache 2 is a reasonable choice -- the only alternative I'd been considering is the Postgres license.

**Comment by t3hmrman** [comment]
Resolved by #17
